app/static/style.css

```diff
@@ -31,33 +31,44 @@ body {
 .profile-modal-fadeIn {
     -webkit-animation-name: profile-modal-fadeIn;
     animation-name: profile-modal-fadeIn;
-    
     -webkit-animation-duration: 1s;
     animation-duration: 1s;
     -webkit-animation-fill-mode: both;
     animation-fill-mode: both;
-  }
-  
-  @keyframes profile-modal-fadeIn {
+}
+
+@keyframes profile-modal-fadeIn {
```

Comment:
Why?

Reply:
This has been done automatically by some extension I used. I disabled it and made sure it is exactly the same as in develop.